bundles/ch.elexis.kassenbuch/src/ch/elexis/buchhaltung/kassenbuch/BuchungsDialog.java

```diff
@@ -32,13 +36,13 @@
 import ch.rgw.tools.TimeTool;
 
 public class BuchungsDialog extends TitleAreaDialog {
-
+	private final ExecutorService executor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors());
 	boolean bType;
 	LabeledInputField liBeleg, liBetrag;
 	CDateTime liDate;
 	Label dDate;
 	Text text;
-	KassenbuchEintrag last, act;
+	KassenbuchEintrag last, act, lastNr;
```

Comment:
Eigentlich wird das `last` Feld nicht mehr verwendet wenn der letzte Eintrag mit `KassenbuchEintrag.lastNr();` in das `lastNr` Feld geholt wird. Bitte `last` entfernen.